### PR TITLE
Add contact section on the mobile version

### PR DESCRIPTION
**Pull request by Kirabo19**
Add contact form 
Finish the mobile portfolio version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,7 +246,28 @@ <h2 class="media_header">LET'S CONNECT</h2>
...
+    <footer class="contact_container">
+      <div>
+        <div class="container flex">
+          <h1 class="feedback">contact me</h1>
+          <p class="msg">if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+        </div>
+        <div>
+          <form action="#" method="post">
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please add a [formspree](https://formspree.io/) action to the this form tag 👍 . It should have a similar format to `https://formspree.io/f/{form_id}`. Your form tag should then look something like `<form action="https://formspree.io/f/{form_id}" method="post">`

File: index.html [hunk 2]
```diff
@@ -246,7 +246,28 @@ <h2 class="media_header">LET'S CONNECT</h2>
...
+            <input type="text" aria-required="true" maxlength="30" name="name" placeholder="patrick rejoice">
+            <input type="email" aria-required="true" maxlength="30" name="email" placeholder="rmjspecial2015@gmail">
+            <textarea aria-required="true" maxlength="30" name="name" placeholder="Write your message here"></textarea>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please make the max length 500. It should look something like this `maxlength="500"` 🏄

File: css/port.css [hunk 3]
```diff
@@ -296,3 +296,82 @@ section .aim ul li {
...
+  margin-top: 10px;
+}
+
+.line {
+  background-color: #0d151e;
+  height: 5px;
+  width: 134px;
+  border-radius: 10px;
+  margin-left: 100px;
+}
```
Comment (Lucas-Masaba) [inline css/port.css]: - [ ] The input fields do not match the design in the Figma template. Please fix your implementation to resemble that of the Figma template 👍 
### Your implementation
image
### Figma
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Kirabo19  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Good pull reequest and comit messages 🏄 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (chrystalme) [reply index.html]: Kindly add action link to your form as specified by the previous reviewer.

File: index.html (hunk 2 above)
Comment (chrystalme) [reply index.html]: Kindly set the maxlength property of the text area to 500 rather than 300. Also, the name of this field should be `message`

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hi @Kirabo19, @rmjspecial  and @chimwemwe007 , 

You have done a good job so far! 👏🏽 👏🏽 👏🏽 
However, there are some issues that you still need to work on to prepare your project for further evaluation

## Suggested changes
_Check the comments under the review._



- [ ] I have a observed that this project has no new commits. It was just submitted. Kindly make changes before you submit this project again for review



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Kirabo19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**